src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -63,10 +62,21 @@ public ApiController (String apiPrefix, TypedPersistence<T> persistence, String
      */
     @Override
     public void bind(final SparkSwagger restApi) {
-        LOG.info("Registering routes and enabling docs for {}", ROOT_ROUTE);
+        ApiEndpoint apiEndPoint = restApi.endpoint(endpointPath(ROOT_ROUTE)
+            .withDescription("Interface for querying and managing '" + classToLowercase + "' entities."),
+            (q, a) -> LOG.info("Received request for '{}' Rest API", classToLowercase)
+        );
+        makeEndPoint(apiEndPoint);
+    }
 
-        restApi.endpoint(endpointPath(ROOT_ROUTE)
-            .withDescription("Interface for querying and managing '" + classToLowercase + "' entities."), (q, a) -> LOG.info("Received request for '{}' Rest API", classToLowercase))
+    /**
+     * Adds a basic implementation of the HTTP CRUD methods that can be overriden by child classes.
```

Comment:
Typo 'overriden' should be 'overridden'

Reply:
This javadoc feels a little sparse. How about something like:

```
This method implements a set of basic HTTP Spark methods (e.g., getOne, getMany, delete) for CRUD operations. It can be overridden by child classes to supplement additional methods through the ApiEndpoint (after supplemental methods are added, be sure to call the super method to add the basic methods).
```

And now that I'm taking a closer look: should `apiEndpoint` just be a field on this class? And I don't think the `makeEndpoint` method needs to return the ApiEndpoint, right?

Reply:
Good catch, it is no longer necessary.
I had the return because I initially thought doing a chain add, e.g.
```java
return super.makeEndPoint()
.get(path(...))
.options(path(...))
```
but we have to call `super` after adding the custom routes. I'm updating the comments in this method accordingly.

Reply:
It don't think `apiEndpoint` can become a field because the parent, `baseEndPoint`, is not available until spark-java calls the `bind` method.